tools/packaging/kernel/configs/fragments/common/cryptsetup.conf

```diff
@@ -0,0 +1,43 @@
+CONFIG_MD=y
```

Comment:
I am unsure the `common` dir is best place for the `cryptsetup.conf` file as it will include dm stuffs for all kernels. Ignore the following idea if you think it is okay that.

I'm not so used to the kernel build scripts of kata, but looking at `build-kernel.sh` it seems the kernel for confidential guest is built with passing `-x tdx|sev` option and their specific configs live under `configs/fragments/x86_64`. There is also the `configs/fragments/build-type` dir which contain configs for when passing `-b` (build type), `-e` (experimental) or `-E` (arch-experimental). I'm wondering if putting our configs under `configs/fragments/build-type/*confidential_containers*` then setting the `build_type` variable when `-x tdx|sev` wouldn't keep it more organized.

Reply:
I changed to use `KATA_BUILD_CC` to control the build for this feature, since we may need test these features without HW TEE. `cryptsetup.conf` is generic for different HW, then I put it under `common/confidential_containers` directory.